src/main/java/graphql/ExecutionResult.java

```diff
@@ -11,6 +11,13 @@
 
     List<GraphQLError> getErrors();
 
-    Map<Object,Object> getExtensions();
+    Map<Object, Object> getExtensions();
```

Comment:
On second thought, In my work we construct our own object to be sent as a response, so we don't use this getExtensions() method.
Nevertheless, make a PR that supports extensions is better than delete this method and maybe forget about it later in the future.

But again, I think this discussion belongs to another PR, because this is not the topic about this PR as i see it. If you want me to remove this method an accept this PR, I would be happy to do so